requirements.txt

```diff
@@ -14,7 +14,7 @@ requirements-parser
 # Pin pytest due to VS Code issue
 pytest==5.0.1
 pytest-cov
-tempeh==0.1.6
+tempeh==0.1.7
```

Comment:
Should we start going with '>=' here?

Reply:
No, because tempeh is in no way backwards compatible and can break the tests here anytime.